easybuild/easyconfigs/n/netCDF-Fortran/netCDF-Fortran-4.4.4-CrayCCE-17.06.eb

```diff
@@ -4,10 +4,11 @@ easyblock = 'ConfigureMake'
 
 name = 'netCDF-Fortran'
 version = "4.4.4"
+versionsuffix = '-no-pkg-config'
```

Comment:
please remove the versionsuffix, this was only useful for the testing and we're not planning to keep two versions of this modules.

Reply:
done! 👍